scripts/experience-decisioning/preview.js

```diff
@@ -214,7 +214,7 @@ async function decorateExperimentPill(overlay) {
       label: config.label,
       description: `
         <div class="hlx-details">
-          ${config.status}${config.audience ? ', ' : ''}${config.audience}${config.variants[config.variantNames[0]].blocks.length ? ', Blocks: ' : ''}${config.variants[config.variantNames[0]].blocks.join(',')}
+          ${config.status}${config.audiences.length ? ', ' : ''}${config.audiences[0]}${config.variants[config.variantNames[0]].blocks.length ? ', Blocks: ' : ''}${config.variants[config.variantNames[0]].blocks.join(',')}
```

Comment:
it would be good to display segmentation information in the pill, so that practitioners know the page is "segmented" and maybe even simulate the different audiences experiences.

Reply:
Agreed. I've been thinking the same and am still exploring the best approach to show this. I'm planning on opening different PRs so we can test those out:
1. separate pill for the audiences and the capability to switch between them
2. enhance the experimentation pill and inject new "variant" entries for every audience combo